[DOCS]Adds the release notes for 8.9.0. (#161909)
elastic/kibana: docs/CHANGELOG.asciidoc

* Integrate Conditional Actions with several Observability rule types {kibana-pull}159522[#159522]
* Adds AAD Fields API {kibana-pull}158516[#158516]
* Adds API to retrieve the `fieldsForAAD` from a given rule type {kibana-pull}158516[#158516]
* Improves the performance of clearing expired snooze schedules {kibana-pull}157909[#157909]
APM::
* Ensure Saved Objects are versionable {kibana-pull}159881[#159881]
* Adds active alerts column for transaction group table {kibana-pull}159552[#159552]
* Adds an OpenAI integration {kibana-pull}158678[#158678]
* Adds Storage explorer improvements {kibana-pull}157303[#157303]
* Adds logic to replace transaction histogram with summary for backwards compatibility {kibana-pull}155714[#155714]
Dashboard::
* Adds design enhancements to the clone experience in Dashboards {kibana-pull}159752[#159752]
* Adds enhancements to the empty state screen in Dashboards {kibana-pull}158496[#158496]
* Adds a query DSL documentation link to filters UI {kibana-pull}156543[#156543]
* Adds a counter displaying the min/max values of the time series counter field in the field popover {kibana-pull}155499[#155499]

> * Adds ability to recover from non-fatal error state {kibana-pull}158087[#158087]

Reviewer: We should mention that this line item applies to the Controls.

"Adds the ability for Controls to recover from non-fatal error states." might work.

Author: Applied change

> * Adds ability to recover from non-fatal error state {kibana-pull}158087[#158087]

Author: ```suggestion
* Adds the ability for Controls to recover from non-fatal error state {kibana-pull}158087[#158087]
```

Discover::

> * Updates Discover sharing capabilities to enable sharing a link to Discover when using ad-hoc (temporary) data views {kibana-pull}154947[#154947]

Reviewer 2: ```suggestion
* Updates Discover sharing capabilities to enable sharing a link when using temporary data views {kibana-pull}154947[#154947]
```
I'd suggest to go with the name in our docs, also it might be redundant to mention Discover a second time

[float]
[[fixes-v8.9.0]]
=== Bug Fixes
Alerting::
* Fixes containment boundaries not being re-fetched when a query changes {kibana-pull}157408[#157408]
* Fixes the charts on Log threshold breached details page {kibana-pull}160321[#160321]
APM::
* Fixes infinite loading of APM alert table {kibana-pull}161134[#161134]
* Fixes other bucket message and sorting {kibana-pull}159919[#159919]
* Fixes percentiles for service transaction metrics {kibana-pull}158913[#158913]
* Fixes stack trace on errors when only available as plain text {kibana-pull}156831[#156831]
Cases::
* Fixes a bug in the alerts table where you cannot create a new case when attaching alerts to a case from the cases modal {kibana-pull}160526[#160526]
* Fixes an issue where the following special characters could not be included in the case tags: `\\():<>"*` because it resulted in a bug where the case would not be displayed in the cases table when filtered for those tags. These characters are now handled correctly and the cases will be shown in the table. {kibana-pull}159815[#159815]
Dashboard::

> * Fixes an issue where it was possible for a URL update (redirecting to the alias) to happen in the middle of building the errored state of the Dashboard {kibana-pull}161043[#161043]

Author: ```suggestion
* Fixes an issue where a Dashboard redirect alias displayed a blank screen instead of redirecting properly {kibana-pull}161043[#161043]
```

> * Fixes an issue where it was possible for a URL update (redirecting to the alias) to happen in the middle of building the errored state of the Dashboard {kibana-pull}161043[#161043]

Reviewer: I would reword this to something like:

`Fixes an issue where a Dashboard alias redirect would show a blank screen instead of redirecting properly.`

Author: Applied change

* Fixes an issue where the time slider would override custom time ranges {kibana-pull}160938[#160938]
* Fixes an issue where if the Reset button is clicked after changing the Title or Description in Panel settings before clicking Save, the Title or Description would revert to previous values {kibana-pull}159430[#159430]
* Fixes dashboard reset when initial state has no controls {kibana-pull}159404[#159404]
Design::
* Accessibility: Make links on job validation step distinguished from surrounding text {kibana-pull}160608[#160608]
* Fixes the range slider rendering issues and performance improvements {kibana-pull}159271[#159271]
Discover::

> * Fixes documents request missing pinned filters {kibana-pull}160693[#160693]

Author: ```suggestion

```

> * Fixes documents request missing pinned filters {kibana-pull}160693[#160693]

Reviewer 2: luckily, we introduced this bug in 8.9 and also fixed it then, so this regression escaped into the wilderness

Reviewer 2: I wanted to day: I didn't escape! 😄
